hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestFSImageWithSnapshot.java

```diff
@@ -23,10 +23,7 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.fs.SafeModeAction;
 import org.apache.hadoop.fs.permission.FsPermission;
-import org.apache.hadoop.hdfs.DFSTestUtil;
-import org.apache.hadoop.hdfs.DFSUtil;
-import org.apache.hadoop.hdfs.DistributedFileSystem;
-import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.apache.hadoop.hdfs.*;
```

Comment:
We should avoid to wildcard import.

Reply:
[remove wildcard import](https://github.com/apache/hadoop/pull/6929/commits/0df56b2fdb6f0e7997565deb9038ef4ab0c33fb1)
Thank you for review